music21/converter/subConverters.py

```diff
@@ -838,27 +838,32 @@ class ConverterMusicXML(SubConverter):
     '''
     registerFormats = ('musicxml', 'xml')
     registerInputExtensions = ('xml', 'mxl', 'mx', 'musicxml')
```

Comment:
we can get rid of "mx" here too -- it was from the days before musicxml had its own extension besides .xml and was only a music21 supported method.  No one has used it for years.